Detectors/TRD/simulation/src/TrapSimulator.cxx

```diff
@@ -2007,14 +1994,16 @@ void TrapSimulator::fitTracklet()
       slope = -slope;
       temp = mult * position;
       position = temp >> 32; // take the upper 32 bits
+
+      position += yoffs;
+
       LOG(debug) << "slope = " << slope;
-      //slope = slope;//this is here as a reference for what was done in run2 //* ndrift) >> ndriftDp) + deflCorr;
-      auto oldpos = position;
+
+      slope = slope + ((deflCorr / ndrift) << ndriftDp);
```

Comment:
This is the deflection correction I am not sure about..

Reply:
It can be adjusted or redone later, might be easier to diagnose further down the pipeline?

Reply:
Agree, would also leave it in for now